website/docs/r/iam_trusted_profiles_link.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile_link"
+description: |-
+  Manages iam_trusted_profile_link.
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile_link
+
+Create, update, and delete an IAM trusted profile link. For more information, about IAM trusted profile, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-link 
+
+## Example usage
+
+```terraform
+resource "ibm_iam_trusted_profile_link" "iam_trusted_profile_link" {
+  cr_type = "cr_type"
+  link = { "crn" : "crn", "namespace" : "namespace" }
+  profile_id = "profile_id"
+}
+```
+
+## Argument reference
+
+Review the argument reference that you can specify for your resource.
+
+* `cr_type` - (Required, Forces new resource, String) The compute resource type. Supported values are VSI, IKS_SA, ROKS_SA.
```

Comment:
VSI, IKS_SA, ROKS_SA.  these values must be in bold string as

**VSI**, **IKS_SA**, **ROKS_SA**.